### ISSUE TITLE
CiviCRM

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/backdrop_upgrade_status#14]
I know it's not a great idea to hard code for a particular module, but given the unique space that CiviCRM holds and the unique way that it needs to be installed, how do people feel about hard coding a check for CiviCRM?

![Screen Shot 2020-12-16 at 3 16 04 PM](https://user-images.githubusercontent.com/871421/102407487-d0d03780-3fb1-11eb-9608-efc509e03dea.jpg)

**Comment by laryn** [comment]
@herbdool or @jenlampton -- any thoughts on this? I think it would be good to roll out the red carpet for any D7 Civi sites that may be having cold feet about D8+ and making this display green is one way we could do that. If I can mix my color metaphors.

**Comment by jenlampton** [comment]
I would be fine hard-coding CiviCRM, or adding in some kind of hook or alter so that any module could add its own Backdrop update status -- or both?

**Comment by yorkshire-pudding** [comment]
I think this is a good idea.  I was a little perplexed that there wasn't a module

**Comment by herbdool** [comment]
@yorkshire-pudding CiviCRM is hosted on its own server, and it a more complicated bundle of its own codebase plus a Backdrop module. So it's not straightforward like modules that are hosted on github and only work with Backdrop.

**Comment by yorkshire-pudding** [comment]
@herbdool Yes, it took me a while to work it out, but I got there.  I haven't attempted it yet as still got lots of other things I'm trying.
